scripts/generators/asciidoc_fields.py

```diff
@@ -19,22 +58,10 @@ def save_asciidoc(file, text):
 
 # Field Index
 
-
+@templated('fields_template.j2')
```

Comment:
I like the 1:1 mapping between the full templates (fields_template.j2 and field_values_template.j2) and their rendered `.asciidoc` counterpart 👍 

I also like that `field_details/` follows the same 1:1 with its `.asciidoc` counterpart, except it's a directory this time :-)

The following comment is for a subsequent PR perhaps; by no means a blocker for this one. Could we eventually have a master template for `field-details.asciidoc` as well, and call out from this master template to sub-templates as needed? Not sure if it's easy to do with Jinja2.

Reply:
I agree it will be more consistent to have the master `field_details.j2` file, and I plan on tackling this soon.

Jinja does have features like [template inheritance](https://jinja.palletsprojects.com/en/2.11.x/templates/#template-inheritance) and [include tags](https://jinja.palletsprojects.com/en/2.11.x/templates/#include) to help simplify here. 😃